dvc/repo/checkout.py

```diff
@@ -8,14 +8,14 @@
 logger = logging.getLogger(__name__)
 
 
-def _cleanup_unused_links(repo):
+def _get_unused_links(repo):
     used = [
         out.fspath
         for stage in repo.stages
         for out in stage.outs
         if out.scheme == "local"
     ]
-    repo.state.remove_unused_links(used)
+    return repo.state.get_unused_links(used)
```

Comment:
TODO:
- [ ] `get_unused_links` returns directory name without `/` at the end

Reply:
Because it is not for presentation. I don't like where it's headed all internal code should not care about presentation, that will complicate things a lot.

Reply:
Yes, I'm not suggesting to change internal functions at all. Just the `_get_unused_links` function inside the `repo/checkout.py`. I'd love to handle it in `command`, but as we need to return dict (as we might need to support `--show-json` later), I feel this is the place to do. 

But, I agree with you. I am not liking this at all, sprinkling logic all over the place to return changes. If you have any better suggestion, please do tell.

Reply:
@Suor, we could return stats without stringifying filepaths (i.e. as PathInfo), and only add the `os.sep` on the deleted files beforehand (on `Repo._checkout()` itself). This way, presentation logic would be on the `cli` side. However, this will divide the logic anyway.